### PR TITLE
KVM - add backuprestore test config

### PR DESCRIPTION
**Pull request by kumudag**
Adding config file for backuprestore test

Signed-off-by : Kumuda G <kumuda@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by kumudag** [comment]
output of --dry-run using backuprestore.cfg

[backuprestore_dryrun.txt](https://github.com/open-power-host-os/tests/files/4305489/backuprestore_dryrun.txt)

File: config/tests/guest/libvirt/backuprestore.cfg [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+mem = 32768
+
+variants:
+    - guest_import:
+        only unattended_install.import.import.default_install.aio_native
+
+    - guest_backuprestore:
+        only virsh.snapshot_dumpxml,virsh.snapshot_par_cur,virsh.snapshot_disk,virsh.snapshot_create_as
+        #Skipping qed 98 test cases ,as these are supported only for libvirt having version 1.1.0
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.attach_img_qed.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
```
Comment (sathnaga) [inline config/tests/guest/libvirt/backuprestore.cfg]: you could filter all qed tests with something like 
```
no virsh.snapshot_disk..attach_img_qed
```
it should match all tests

File: config/tests/guest/libvirt/backuprestore.cfg [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        no virsh.snapshot_disk.no_delete.negative_test.no_pool.network_disk.inactive.snapshot_from_xml.disk_external.disk_only.memory_none
+        #Skipping test case due to bug (https://bugzilla.redhat.com/show_bug.cgi?id=1083345)
+        no virsh.snapshot_disk.delete_test.positive_test.no_pool.attach_img_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.dir_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.dir_pool.attach_img_qcow2.v_qcow2v3.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_pool.attach_img_qcow2.v_qcow2v3.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_gluster_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
```
Comment (sathnaga) [inline config/tests/guest/libvirt/backuprestore.cfg]: same here,  something like this should cover all `memory_no` tests, choose according to your needs, `virsh.snapshot_disk..memory_no`

File: config/tests/guest/libvirt/backuprestore.cfg [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.no_snapshot_attr.multi_disk_external
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.disk_only_spec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.check_libvirtd_log 
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.quiesce_with_diskonly.with_diskspec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_default
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.raw
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.xz
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.lzop
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.gzip
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.bzip2
```
Comment (sathnaga) [inline config/tests/guest/libvirt/backuprestore.cfg]: same

File: config/tests/guest/libvirt/backuprestore.cfg [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.name_with_double_dash
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.quiesce_with_diskonly.no_diskspec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.atomic_with_diskonly
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.no_metadata
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.no_metadata_with_diskonly
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.multi_snapshots
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.with_snapshot_attr.disk_only_spec
+        no virsh.snapshot_create_as..quiesce_without_diskonly #Not supported- guest agent
+        no virsh.snapshot_create_as.positive_tests.disk_only_spec #Needs fix, hardcoded as vda
+        no virsh.snapshot_create_as.positive_tests.reuse_external #Needs fix, hardcoded as vda
```
Comment (sathnaga) [inline config/tests/guest/libvirt/backuprestore.cfg]: I hope these are internal comments, let remove them and have it locally tracked and enable these tests with furture PRs..

**Comment by kumudag** [comment]
Latest dry-run output..
[backuprestore_dryrun.txt](https://github.com/open-power-host-os/tests/files/4306293/backuprestore_dryrun.txt)

**Pull request merged**